### DISCUSSION TITLE
Underlined text not displayed on Firefox

### DISCUSSION THREAD

**Discussion by Alhadis** [community/community#72687] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

On Chrome, rendered code spans with the class `.pl-corl` are correctly underlined. On Firefox, however, the underline disappears. Compare the appearance of [line 17 of this file](https://github.com/Alhadis/language-etc/blob/master/samples/lists/scope-previews.nanorc#L17) between both browsers.

I can't for the life of me fathom why GitHub's front-end is using generated content (i.e., pseudo-elements in CSS parlance) to display lines of syntax highlighted code. But a quick fix for this would be to amend the selector of the ruleset that applies text-underlining (this is only a Q&D fix; tweak selector as necessary):

```diff
-.pl-corl {
+.pl-corl, .pl-corl > [data-code-text]::before {
 	text-decoration: underline;
 	color: var(--color-prettylights-syntax-constant-other-reference-link);
 }
```

**Comment by iprattham** [comment]
The issue you're describing involves underlining rendered code spans with the class .pl-corl on Chrome and Firefox, and it seems to be related to pseudo-elements in CSS.

The code you provided suggests a possible fix by extending the selector to include the pseudo-elements that are causing the issue on Firefox. In other words, you're not just targeting elements with the class .pl-corl but also the ::before pseudo-element within those elements or their children.

Here's the proposed fix:

```
css
.pl-corl, .pl-corl > [data-code-text]::before {
  text-decoration: underline;
  color: var(--color-prettylights-syntax-constant-other-reference-link);
}
```

This CSS rule targets elements with the class .pl-corl and any ::before pseudo-elements within those elements or their children, and it adds the text-decoration property to underline the text.

To apply this fix:

1. Edit the CSS for your website or project.
2. Insert the provided CSS rule at the appropriate place in your stylesheet.

This fix should ensure that the underline is consistently applied to the .pl-corl elements and their pseudo-elements on both Chrome and Firefox.

**Reply by Alhadis** [reply]
@iprattham This is an issue with GitHub.com's front-end, and not a project I have any influence over (nor is the aforementioned front-end open-sourced to the public). I was under the impression that GitHub solicits bug reports from users in the form of a discussion on this org. Has something changed? Is there somewhere else I should be reporting this to?

**Comment by Alhadis** [comment]
Alright, I've found the correct outlet to report site issues to, and I've filed a ticket for this issue. See [`#2479853`](https://support.github.com/ticket/2479853).

Closing this to avoid diluting futher discussion on the matter.
